cmake/oneDPLWindowsIntelLLVMApply.cmake

```diff
@@ -37,5 +39,5 @@ endif()
 if (INTELLLVM_WIN_STD_IGNORE_FIX)
     include(Compiler/CMakeCommonCompilerMacros)
     # Setting c++ standard default to 14, and enabling usage of CMAKE_CXX_STANDARD
```

Comment:
```suggestion
    # For IntelLLVM versions greater or equal to 2020, setting c++ standard default to 14, and enable usage of CMAKE_CXX_STANDARD
```

Reply:
Seems good to me. I prefer to apply it independent of this patch.

Reply:
Sure I'll make a PR.

Reply:
https://github.com/oneapi-src/oneDPL/pull/1695